crates/pathfinder/src/sequencer.rs

```diff
@@ -89,6 +89,18 @@ pub trait ClientApi {
         contract_definition: ContractDefinition,
         token: Option<String>,
     ) -> Result<reply::add_transaction::DeployResponse, SequencerError>;
+
+    #[allow(clippy::too_many_arguments)]
```

Comment:
Is it OK to ask clippy to let it go this one time? :)

Reply:
Yes.

Reply:
I think we've already stepped into this realm with some other `fn`s, so yes.

Reply:
Ack